DependencyInjection/Factory/Loader/FileSystemLoaderFactory.php

```diff
@@ -61,4 +63,18 @@ public function addConfiguration(ArrayNodeDefinition $builder)
                 ->end()
             ->end();
     }
+
+    /**
+     * @param string $reference
+     *
+     * @return Reference
+     */
+    private function createLocatorReference($reference)
+    {
+        return new Reference(
+            sprintf('liip_imagine.binary.locator.%s', $reference),
+            ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE,
+            SymfonyFramework::hasDefinitionSharedToggle()
```

Comment:
The reason I left it like this is because that will always be true for Symfony `>=2.8`, and therefore it doesn't matter that it's ignored. We only care about it when it is `false` on Symfony `<2.8`. Otherwise, the extra parameter can be, and is, safely ignored (the PHP interpreter allows and ignores any extra parameters). Do you really think it's necessary to add a switch?